examples/fluent-bit/amazon-opensearch-serverless/README.md

```diff
@@ -0,0 +1,21 @@
+### FireLens Example: Logging to Amazon OpenSearch Serverless Service with Fluent Bit
```

Comment:
It's just "Amazon OpenSearch Serverless" (not "Service")

Reply:
Updated! Thanks